Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/event/model/Document.java

```diff
@@ -419,7 +421,7 @@ public void setContentType(String contentType) {
      * @return the documentType
      */
     public String getDocumentType() {
-        return documentType;
+        return (documentType == null) ? documentClassCode : documentType;
```

Comment:
Its basically a workaround for now for the demo, eventually when the Demo Adapters are updated to retrieve both documettype and documentclasscode, we will change it.